open_in_mpv/open_in_mpv.py

```diff
@@ -4,19 +4,36 @@
 from typing import Any, BinaryIO, Callable, Final, Mapping, TextIO, cast
 import json
 import os
+import random
 import socket
 import struct
 import subprocess as sp
 import sys
 import tempfile
+import time
 
 from loguru import logger
 import click
 import xdg.BaseDirectory
 
 from .constants import IS_MAC, IS_WIN, MACPORTS_BIN_PATH
 
+try:
+    from win32file import GENERIC_READ, GENERIC_WRITE, OPEN_EXISTING, CreateFile, WriteFile
```

Comment:
These should be in alphabetical order according to my isort rules.

Reply:
this was the result of `yarn format` :shrug: